src/services/services.ts

```diff
@@ -375,6 +375,68 @@ namespace ts {
 
         return documentationComment;
 
+        function getCommentFromJsDocTag(tag: JSDocTag, rangeEnd: number, sourceFile: SourceFile) {
+            const text = sourceFile.text;
+
+            const parts: string[] = [];
+            let start = tag.end + 1;
+
+            // Eat any leading ignored characters
+            while (start < rangeEnd) {
+                const ch = text.charCodeAt(start);
+                if (ch === CharacterCodes.asterisk || isWhiteSpace(ch)) {
+                    start++;
+                }
+                else if (ch === CharacterCodes.at) {
+                    // Most likely a malformed tag
+                    return undefined;
+                }
+                else {
+                    break;
+                }
+            }
+
+            let i: number;
+            for (i = start; i < rangeEnd; i++) {
+                const ch = text.charCodeAt(i);
+                if (isLineBreak(ch)) {
+                    // Line break, push any non-empty content so far
+                    if (start !== -1 && start !== i) {
+                        parts.push(text.substr(start, i - start));
+                    }
+
+                    // Keep scanning until we hit either a a non-ws non-* char
+                    while (i < rangeEnd) {
+                        i++;
+                        const nextCh = text.charCodeAt(i);
+                        if ((nextCh === CharacterCodes.asterisk) || (isWhiteSpace(nextCh) || isLineBreak(nextCh))) {
+                            // Eat *s and whitespace
+                        }
+                        else if (nextCh === CharacterCodes.at) {
+                            // Abort, we're running into another malformed jsdoc tag

```

Comment:
Only if the @ occurs at the start of a line (which is basically indistinguishable from a malformed comment)